Practice in php/colorcalculator.php

```diff
@@ -0,0 +1,42 @@
+<html>
+
+		<form action= "">
+			<select name='ColorSelector1' class='ColorSelector1'>
+					<option value=''> Select</option>
+					<option value='Red'> Red </option>
+					<option value='Blue'> Blue </option>
+					<option value='Yellow'> Yellow </option>
+			</select>
+			<select name='ColorSelector2'>
+					<option value=''> Select</option>
+					<option value='Red'> Red </option>
+					<option value='Blue'> Blue </option>
+					<option value='Yellow'> Yellow </option>
+			</select>
+			<input type='submit' Value='Result'/>
+
+</html>
+
+<?php
+
+
+if(isset($_REQUEST['ColorSelector1']) && $_REQUEST['ColorSelector1']=='Red' || isset($_REQUEST['ColorSelector2']) && $_REQUEST['ColorSelector2']=='Red'){
```

Comment:
It looks like this wasn't totally finished. That's fine, but let me know if you want to talk through how you could handle all the logic in these if statements.